src/main/resources/alma/index-config.json

```diff
@@ -382,7 +382,12 @@
                      "index" : "false"
                   },
                   "callNumber" : {
-                     "type" : "keyword"
+                     "type" : "keyword",
+                     "index" : "false"
```

Comment:
Sorry, but I have to "veto" this change. In DigiBib, we need to be able to search for call numbers.

There might be a different solution coming up in the context of #1614, but for now it's indispensable.